README.md

```diff
@@ -1,42 +1,42 @@
 # Haven
 
-[Haven](https://havenweb.org) is a simple blog application (formerly called SimpleBlog) built with Ruby on Rails, some motivating philosophies:
+[Haven](https://havenweb.org) is a simple blog application (formerly called SimpleBlog) built with Ruby on Rails, based on some motivating philosophies:
```

Comment:
How about "here are" instead of "based on"

Reply:
It would still be a comma splice, then.

Reply:
We can split it with a period, make "Here are" the start of a new sentence.  Does that work?  Really appreciate your help cleaning up the readme!